### PR TITLE
Add IdentityCredential to Credential Type Registry

### PR DESCRIPTION
**Pull request by npm1**
Fixes https://github.com/fedidcg/FedCM/issues/301

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/pull/201.html" title="Last updated on Jul 8, 2022, 1:43 PM UTC (fa4b0c8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webappsec-credential-management/201/878530d...fa4b0c8.html" title="Last updated on Jul 8, 2022, 1:43 PM UTC (fa4b0c8)">Diff</a>

### CODE REVIEW

**Review by johannhof — CHANGES_REQUESTED** [review]
Had a few comments. Personally I'm not a fan of whitespace-cleanup on unrelated lines but I'll leave that to the editor's preference. :)

Note that I think I'll want to defer merging to @nsatragno as editor. It seems like this can wait until she's back, right?

**Review by johannhof — APPROVED** [review]
Sorry, meant to approve but let's wait on Nina.

File: index.bs [hunk 1]
```diff
@@ -96,6 +99,11 @@ spec:css-syntax-3;
 </pre>
 <pre class="biblio">
 {
+  "FEDCM": {
+    "authors": [ "Sam Goto" ],
+    "href": "https://github.com/fedidcg/FedCM",
```
[the remark this answers is not in the document]
Comment (npm1) [reply index.bs]: Yea probably! I just copied the syntax from WEB-LOGIN, which seems incorrect.

File: index.bs [hunk 2]
```diff
@@ -326,6 +334,13 @@ spec:css-syntax-3;
         <td>This specification: [[#federated]]</td>
         <td><a href="https://www.w3.org/2011/webappsec/">W3C</a></td>
     </tr>
+    <tr>
+        <td>identity</td>
+        <td>identity</td>
+        <td>{{IdentityCredential}}</td>
+        <td>[[FEDCM]]</td>
+        <td><a href="https://www.w3.org/community/fed-id/">W3C</a></td>
```
[the remark this answers is not in the document]
Comment (npm1) [reply index.bs:342]: I am following the example from the above which says W3C even though it links to WebAppSec specifically... Not sure either!

**Review by npm1 — COMMENTED** [review]
Ah yea I didn't get rid of the whitespaces on purpose, my editor did that automatically heh... There was a lot of them! So I remade the patch to be only the specific change on the title.

File: index.bs (hunk 2 above)
[the remark this answers is not in the document]
Comment (johannhof) [reply index.bs:342]: I don't think this is super important but I have a slight preference for FedID CG given that the spec says WebAppSec will "notify the requestor" for updates, so more precise seems better. We'll just have to update this once it moves out of incubation...

**Review by nsatragno — APPROVED** [review]
LGTM

Thank you @johannhof for the first review!

**Pull request merged**